tests/test_question.py

```diff
@@ -51,17 +52,19 @@ def test_skipping_of_skipping_of_questions():
         inp.close()
 
 
-def test_async_ask_question():
-    loop = asyncio.new_event_loop()
+if is_prompt_toolkit_3():
```

Comment:
I think there is a pytest primitive that does this kind of filtering. (I think using an annotation)

Reply:
@tmbo Thanks - I didn't know this. I've updated it.